File: ido_sdk_flutter/ido_sdk/protocol_sdk/protocol_channel/ios/Classes/Model/IDOContactReviseTimeModel.swift
//
//  IDOContactReviseTimeModel.swift
//  protocol_channel
//
//  Created by hc on 2023/10/12.
//

import Foundation
/// Get firmware local contact file modification time event number
@objcMembers
public class IDOContactReviseTimeModel: NSObject, IDOBaseModel {
    /// 0: No need to send contact file
    /// 1: Need to send contact data
    public var result: Int

    enum CodingKeys: String, CodingKey {
        case result
    }

    public init(result: Int) {
        self.result = result
    }

    public func toJsonString() -> String? {
        if let jsonData = try? JSONEncoder().encode(self) {
            let jsonString = String(data: jsonData, encoding: .utf8)
            return jsonString
        }
        return nil
    }
}

File: ido_sdk_flutter/ido_sdk/protocol_sdk/protocol_channel/android/src/main/kotlin/com/idosmart/model/IDOContactReviseTimeModel.kt
//
//  IDOContactReviseTimeModel.kt
//  protocol_channel
//
//  Created by hedongyang on 2023/10/30.
//

package com.idosmart.model

import com.google.gson.GsonBuilder
import com.google.gson.annotations.SerializedName

///
/// Get firmware local contact file modification time event number

open class IDOContactReviseTimeModel(result: Int) : IDOBaseModel {

    /// 0: No need to send contact file
    /// 1: Need to send contact data
    @SerializedName("result")
    var result: Int = result


    

    override fun toJsonString(): String {
        return GsonBuilder().create().toJson(this).toString()
    }
}
    